extension/src/popup/basics/AccountSubview.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
```

Comment:
now that i'm adding a couple more of these little "subviews", took the time to consolidate some duplicative code while creating the new views